data/data/bootstrap/files/usr/local/bin/node-image-overlay.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -euo pipefail
+
+ostree_checkout=/ostree/repo/tmp/node-image
+if [ ! -d "${ostree_checkout}" ]; then
+    ostree_checkout=/var/ostree-container/checkout
+fi
+
+# keep /usr/lib/modules from the booted deployment for kernel modules
+mount -o bind,ro "/usr/lib/modules" "${ostree_checkout}/usr/lib/modules"
+mount -o rbind,ro "${ostree_checkout}/usr" /usr
+rsync -a "${ostree_checkout}/usr/etc/" /etc
+
+# reload the new policy
+semodule -R
```

Comment:
Should we `systemctl daemon-reexec` here to be safe?

Reply:
We could, but I'd rather like to wait and see what kind of issues we hit before pre-emptively doing that. This may inform other things in this approach.